From ea0cf3e2e2bc399068300efd577df1cc37b2c2a2 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal.vasilek@nic.cz>
Date: Thu, 15 Dec 2022 11:18:00 +0100
Subject: [PATCH] apr-util: disable parallel build

Build reliably fails with -j20

    crypto/apr_passwd.c:200:1: fatal error: error closing -: Broken pipe
      200 | }
          | ^
    compilation terminated.

Signed-off-by: Michal Vasilek <michal.vasilek@nic.cz>
---
 libs/apr-util/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/libs/apr-util/Makefile b/libs/apr-util/Makefile
index 54c0a94c2..ccc45872a 100644
--- a/libs/apr-util/Makefile
+++ b/libs/apr-util/Makefile
@@ -21,8 +21,6 @@ PKG_LICENSE_FILES:=LICENSE
 
 PKG_CPE_ID:=cpe:/a:apache:apr-util
 
-PKG_BUILD_PARALLEL:=1
-
 PKG_CONFIG_DEPENDS := \
 	CONFIG_PACKAGE_libaprutil-crypto-openssl \
 	CONFIG_PACKAGE_libaprutil-dbd-mysql \
-- 
2.39.0

